server/routes/varyingLicences/handlers/policyChange.test.ts

```diff
@@ -0,0 +1,186 @@
+import { Request, Response } from 'express'
+import LicenceService from '../../../services/licenceService'
+import ConditionService from '../../../services/conditionService'
+import PolicyChangeRoutes from './policyChange'
+import { Licence, LicenceConditionChange } from '../../../@types/licenceApiClientTypes'
+
+const conditionService = new ConditionService(null) as jest.Mocked<ConditionService>
+const licenceService = new LicenceService(null, null, null, conditionService) as jest.Mocked<LicenceService>
+
+jest.mock('../../../services/licenceService')
+jest.mock('../../../services/conditionService')
+
+describe('Route handlers', () => {
+  const handler = new PolicyChangeRoutes(licenceService, conditionService)
+  let req: Request
+  let res: Response
+
+  licenceService.getParentLicenceOrSelf.mockResolvedValue({ id: 1, version: 'version', typeCode: 'AP_PSS' } as Licence)
+
+  let condition1 = {
```

Comment:
Apologies, I did look into this but I later pass in conditions1-4 as mocked responses/ session objects, which I would need to redefine if I were to create a new variable in the individual tests. It can definitely be done, but I did it this way to avoid repetition of code.